<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nsLogin</class>
 <widget class="QDialog" name="nsLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>92</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>nsLogin</string>
  </property>
  <widget class="QGroupBox" name="grpUserInfo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>270</width>
     <height>87</height>
    </rect>
   </property>
   <property name="title">
    <string>User Information</string>
   </property>
   <widget class="QLineEdit" name="nsBaseDomain">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>167</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="nsUsername">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>167</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="nsPassword">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>167</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblBaseDomain">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Base Domain</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblUsername">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPassword">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
